lib/vsc/administration/slurm/sacctmgr.py

```diff
@@ -26,6 +27,7 @@
 from vsc.administration.slurm.scancel import create_remove_user_jobs_command
 
 SLURM_SACCT_MGR = "/usr/bin/sacctmgr"
+SLURM_SACCT = "/usr/bin/sacct"
```

Comment:
make a separate module for `sacct`; this is `sacctmgr` only

Reply:
OK

Reply:
do you want all the same boilerplate in `sacct.py` as in `sacctmgr.py`? might lead to some copy pasta.